c7n/resources/securityhub.py

```diff
@@ -552,6 +552,18 @@ def format_resource(self, r):
         raise NotImplementedError("subclass responsibility")
 
 
+class DescribePostFinding(PostFinding):
+    """newer security hub resources use default describe format.
+
+    ie no more brain dead lossy conversions with high manual effort.
```

Comment:
Perhaps this comment should be expressed more positively?

"Allows for high fidelity information recording with low effort." ?

Reply:
perhaps, having fought through many of these high effort mappings before, i'm a little to scarred to feel polite wrt to the service. as a reference point aws ops center was a single days effort that provides significantly more coverage, more features to end users. security hub is mostly a marketing success story, but as an integrator or api users its clear that its a pretty substandard for its own use cases, because they choose arbitrary mappings for each resource with data loss, and a loss of security information.